source/guide_healthchecks.rst

```diff
@@ -0,0 +1,742 @@
+Slack, Signal, ...) if your job didn't execute on time. To track execution HTTP request or mails may be used.
+
+It offers a web interface to manage your jobs to be monitored.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`Python <lang-python>`
+  * :manual:`MySQL <database-mysql>` or :manual:`PostgreSQL <database-postgresql>` (not used in this guide)
+  * :manual:`supervisord <daemons-supervisord>`
+  * :manual:`domains <web-domains>`
+
+License
+=======
+
+All relevant legal information can be found here
+
+  * https://github.com/healthchecks/healthchecks/blob/master/LICENSE
+
+Prerequisites
+=============
+
+First of all we create a new :manual:`MySQL <database-mysql>` database, that our Healthchecks instance will use:
+
+::
+
+ [isabell@stardust ~]$ mysql -e "CREATE DATABASE isabell_healthchecks"
+ [isabell@stardust ~]$
+
+If you'd like to use :manual:`PostgreSQL <database-postgresql>` instead. Please see the respective
+:manual:`manual entry <database-postgresql>`.
+
+.. include:: includes/my-print-defaults.rst
+
+Second we create a :manual_anchor:`new mailbox <mail-mailboxes/#setup-a-new-mailbox>`, that we'll use to send alerts to our private email
+address if one of our jobs failed to run. You'll have to interactively provide a password. Please choose a secure one and remember the
+password, you'll need it later.
+
+::
+
+ [isabell@stardust ~]$ uberspace mail user add healthchecks
+ Enter a password for the mailbox:
+ Please confirm your password:
+ New mailbox created for user: 'healthchecks', it will be live in a few minutes...
+ [isabell@stardust ~]$
+
+Third of all your URL needs to be setup:
+
+.. include:: includes/web-domain-list.rst
+
+
+We're using :manual:`Python <lang-python>` in the currently latest version 3.11:
+
+::
+
+ [isabell@stardust ~]$ python3.11 --version
+ Python 3.11.7
+ [isabell@stardust ~]$
+
+Installation
+============
+
+Download the source
+-------------------
+
+In the following steps we'll create a separate folder for this app and future apps, download the :manual:`Python <lang-python>` source code
+from GitHub and select the latest version using git tags:
+
+.. code-block:: console
+ :emphasize-lines: 21
+
+ [isabell@stardust ~]$ mkdir ~/apps
+ [isabell@stardust ~]$ cd ~/apps
+ [isabell@stardust ~]$ git clone https://github.com/healthchecks/healthchecks.git
+ Cloning into 'healthchecks'...
+ remote: Enumerating objects: 28393, done.
+ remote: Counting objects: 100% (3677/3677), done.
+ remote: Compressing objects: 100% (448/448), done.
+ remote: Total 28393 (delta 3420), reused 3349 (delta 3229), pack-reused 24716
+ Receiving objects: 100% (28393/28393), 22.57 MiB | 14.09 MiB/s, done.
+ Resolving deltas: 100% (22120/22120), done.
+ [isabell@stardust ~]$ cd healthchecks
+ [isabell@stardust ~]$ git tag
+ 1.0.2
+ v1.0
+ v1.0.1
+ v1.0.2
+ [...]
+ v2.9.2
+ v3.0
+ v3.0.1
+ v3.2
+ v3.1
+ [isabell@stardust ~]$ git checkout v3.2
+ Note: switching to 'v3.2'.
```

Comment:
Do you want me to keep all the log output from that command? Or should I somehow abbreviate it?

Reply:
In this case it might be nice to keep the whole output, someone who has never done this gets an idea of what is happening. I let that up to you though, abbreviating it would be fine too.